domain-rrd/rrd-workflows/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -2,13 +2,7 @@
 
 import static java.util.Objects.isNull;
 
-import gov.va.vro.model.rrd.AbdBloodPressure;
-import gov.va.vro.model.rrd.AbdBpMeasurement;
-import gov.va.vro.model.rrd.AbdCondition;
-import gov.va.vro.model.rrd.AbdEvidence;
-import gov.va.vro.model.rrd.AbdMedication;
-import gov.va.vro.model.rrd.AbdProcedure;
-import gov.va.vro.model.rrd.ServiceLocation;
+import gov.va.vro.model.rrd.*;
```

Comment:
Will this cause style checker's complaint?

Reply:
Yes, I will fix it